lib/integrity/integrity.go

```diff
@@ -10,6 +10,9 @@ var (
 	// ErrNotConfigured is reported when integrity check is not configured
 	// in the command context.
 	ErrNotConfigured = errors.New("integrity check is not configured")
+	// integrityCheckPeriod is a flag enables periodic integrity checks.
+	// The default period is 1 day.
+	integrityCheckPeriod = 24 * 60 * 60
```

Comment:
```suggestion
	// integrityCheckPeriod is a flag enables periodic integrity checks.
	// The default period is 1 day.
	integrityCheckPeriod = 24 * 60 * 60
```
The default value is not used anywhere.

Reply:
Deleted it.

Reply:
But we need the default value :(

Reply:
The default value will be set in `tt-ee` in implementation function of creating this flag. In this edition this flag cannot be used, it is initing by nil value which is 0.

Reply:
Okay, I'll wait then for a patch for `tt-ee`.

Reply:
To be honest, I don't see the test here )